tests/core/filter-utils/test_filter_helpers.py

```diff
@@ -308,38 +311,38 @@ def test_check_if_to_block_match(block_number, _type, to_block, expected):
         # length 1 matches
-        (TOPICS_EMPTY, FILTER_MATCH_ANY_ONE, False),
-        (TOPICS_ONLY_A, FILTER_MATCH_ANY_ONE, True),
-        (TOPICS_ONLY_B, FILTER_MATCH_ANY_ONE, True),
-        (TOPICS_ONLY_C, FILTER_MATCH_ANY_ONE, True),
-        (TOPICS_EMPTY, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_EMPTY, FILTER_MATCH_ONLY_B, False),
-        (TOPICS_EMPTY, FILTER_MATCH_ONLY_C, False),
-        (TOPICS_ONLY_A, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_ONLY_B, FILTER_MATCH_ONLY_B, True),
-        (TOPICS_ONLY_C, FILTER_MATCH_ONLY_C, True),
-        (TOPICS_ONLY_B, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_ONLY_C, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_ONLY_A, FILTER_MATCH_ONLY_B, False),
-        (TOPICS_ONLY_C, FILTER_MATCH_ONLY_B, False),
-        (TOPICS_ONLY_A, FILTER_MATCH_ONLY_C, False),
-        (TOPICS_ONLY_B, FILTER_MATCH_ONLY_C, False),
-        (TOPICS_A_A, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_A_B, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_A_C, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_A_B_C, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_A_C_B, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_B_A, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_B_C, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_B_A_C, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_B_C_A, FILTER_MATCH_ONLY_A, False),
+        (TOPICS_EMPTY, FILTER_MATCH_ONE_OR_MORE, False),
+        (TOPICS_ONLY_A, FILTER_MATCH_ONE_OR_MORE, True),
+        (TOPICS_ONLY_B, FILTER_MATCH_ONE_OR_MORE, True),
+        (TOPICS_ONLY_C, FILTER_MATCH_ONE_OR_MORE, True),
+        (TOPICS_EMPTY, FILTER_MATCH_A, False),
+        (TOPICS_EMPTY, FILTER_MATCH_B, False),
+        (TOPICS_EMPTY, FILTER_MATCH_C, False),
+        (TOPICS_ONLY_A, FILTER_MATCH_A, True),
+        (TOPICS_ONLY_B, FILTER_MATCH_B, True),
+        (TOPICS_ONLY_C, FILTER_MATCH_C, True),
+        (TOPICS_ONLY_B, FILTER_MATCH_A, False),
+        (TOPICS_ONLY_C, FILTER_MATCH_A, False),
+        (TOPICS_ONLY_A, FILTER_MATCH_B, False),
+        (TOPICS_ONLY_C, FILTER_MATCH_B, False),
+        (TOPICS_ONLY_A, FILTER_MATCH_C, False),
+        (TOPICS_ONLY_B, FILTER_MATCH_C, False),
+        (TOPICS_A_A, FILTER_MATCH_A, True),
+        (TOPICS_A_B, FILTER_MATCH_A, True),
+        (TOPICS_A_C, FILTER_MATCH_A, True),
+        (TOPICS_A_B_C, FILTER_MATCH_A, True),
+        (TOPICS_A_C_B, FILTER_MATCH_A, True),
+        (TOPICS_B_A, FILTER_MATCH_A, False),
+        (TOPICS_B_C, FILTER_MATCH_A, False),
+        (TOPICS_B_A_C, FILTER_MATCH_A, False),
+        (TOPICS_B_C_A, FILTER_MATCH_A, False),
         # length 2 matches
-        (TOPICS_EMPTY, FILTER_MATCH_ANY_TWO, False),
-        (TOPICS_A_A, FILTER_MATCH_ANY_TWO, True),
-        (TOPICS_A_B, FILTER_MATCH_ANY_TWO, True),
-        (TOPICS_ONLY_A, FILTER_MATCH_ANY_TWO, False),
-        (TOPICS_ONLY_B, FILTER_MATCH_ANY_TWO, False),
-        (TOPICS_ONLY_C, FILTER_MATCH_ANY_TWO, False),
+        (TOPICS_EMPTY, FILTER_MATCH_TWO_OR_MORE, False),
```

Comment:
Same assumption here.  That a None value matches any topic that exists at that position, but not an absent topic.